pkgs/top-level/all-packages.nix

```diff
@@ -9608,7 +9608,8 @@ in
 
   ### SHELLS
 
-  runtimeShell = "${runtimeShellPackage}${runtimeShellPackage.shellPath}";
+  runtimeShell = if stdenv.targetPlatform.isAndroid then "${androidRuntimeShellPackage}/ash" else "${runtimeShellPackage}${runtimeShellPackage.shellPath}";
```

Comment:
```suggestion
  runtimeShell = if stdenv.targetPlatform.isAndroid
    then "${androidRuntimeShellPackage}/ash"
    else "${runtimeShellPackage}${runtimeShellPackage.shellPath}";
```

Reply:
Done.

Reply:
It looks like you didn't so reformat it?

Reply:
It also would be much better to leave it as bash if at all possible. This will break a lot off thing.